src/sentry/sentry_apps/external_requests/select_requester.py

```diff
@@ -56,10 +57,23 @@ def run(self) -> dict[str, Any]:
                     "project_slug": self.project_slug,
                     "uri": self.uri,
                     "error_message": str(e),
+                    "url": url,
                 },
             )
+            raise
```

Comment:
now that we aren't swallowing this exception, do we gracefully handle it?

Reply:
Good pt, I just added a distinction to check now, if we get ApiError -> we return a message about validation failure and if the request to the webhook goes wrong we return a somewhat generic message and log the error.

https://github.com/getsentry/sentry/pull/79377/files#diff-7e1a5ec942c78dda796710a42c8a675441de74e2bdfa4653344b6fd711e73dcfR40-R51